Content.Client/Preferences/UI/HumanoidProfileEditor.xaml.cs

```diff
@@ -477,34 +471,26 @@ public sealed partial class HumanoidProfileEditor : Control
             #endregion FlavorText
 
             #region Dummy
+
+            _previewRotateLeftButton.OnPressed += _ =>
+            {
+                _previewRotation += 7;
+                _needUpdatePreview = true;
+            };
+            _previewRotateRightButton.OnPressed += _ =>
+            {
+                _previewRotation += 1;
```

Comment:
cant you use the Direction.TurnCw/TurnCcw methods instead (and not cast to int)?

Reply:
I missed this, nice

Reply:
Because there used `-1` this methods can go below zero that will cause problems, so I use `usize`

Reply:
this looks really bad imo, just store `Direction` and use the methods, i dont see why that should cause problems. going below zero is fine, `-1 % 8` is 7